src/Esfa.Vacancy.Azure.Resources/azuredeploy.json

```diff
@@ -315,14 +329,16 @@
               {"name": "CacheConnectionString", "value": "[parameters('CacheConnectionString')]"},
               {"name": "UseSandboxServices", "value": "yes"},
               {"name": "InstanceName", "value": "[variables('sandboxName')]"},
-              {"name": "CacheReferenceDataDuration", "value": "[parameters('CacheReferenceDataDuration')]"}
+              {"name": "CacheReferenceDataDuration", "value": "[parameters('CacheReferenceDataDuration')]"},
```

Comment:
Why would the Manage app need the ElasticSearch cred's? @dev4valtech @smneal

Reply:
If it doesn't it can be taken out. Let me know.

Reply:
yep it's not needed for Manage API. Only for the Register/Get API

Reply:
That's done.

Reply:
👍